react/components/InfoCard/index.js

```diff
@@ -168,6 +176,32 @@ const InfoCard = ({
 
   const subheadClasses = `${handles.infoCardSubhead} t-body mt6 c-on-base ${alignToken} mw-100`
 
+  const infoCardRef = useRef(null)
+  
+  const { push } = usePixel()
+
+  const promotionEventData =
+    analyticsProperties === 'provide'
+      ? {
+          id: promotionId,
+          name: promotionName,
+          position: promotionPosition,
+        }
+      : undefined
+   
+  useOnView({
+    ref: infoCardRef,
+    onView: () => {
+      if (analyticsProperties === 'none') return
```

Comment:
Very small nitpick, but we prefer to always write our `if`s as a block to make it easier to read 😄 

```suggestion
      if (analyticsProperties === 'none') {
        return
      }
```

Reply:
Done.